packages/core/src/routes/sign-in-experience/custom-ui-assets/index.openapi.json

```diff
@@ -0,0 +1,42 @@
+{
+  "tags": [
+    {
+      "name": "Custom UI assets",
+      "description": "Endpoints for uploading custom UI assets for the sign-in experience. Users can upload a zip file containing custom HTML, CSS, and JavaScript files to replace and fully customize the sign-in experience."
+    },
+    { "name": "Cloud only" },
+    { "name": "Dev feature" }
+  ],
+  "paths": {
+    "/api/sign-in-exp/default/custom-ui-assets": {
+      "post": {
+        "summary": "Upload custom UI assets",
+        "description": "Upload a zip file containing custom HTML, CSS, and JavaScript and replace the default sign-in experience with the custom UI assets.",
```

Comment:
how can we disable custom ui?

Reply:
Use `PATCH api/sign-in-exp` api and set `customUiAssets` to null